OMCompiler/Compiler/Template/SimCodeTV.mo

```diff
@@ -285,7 +286,19 @@ package SimCodeVar
     record INTERNAL end INTERNAL;
```

Comment:
What is `NONECAUS` and `INTERNAL`?

Reply:
NONECAUS means No causality (None), Internal means "internal" in OpenModelica simulation, But for FMUs it is either Local or parameters or calculatedParameters, So for OpenModelica the default internal should be "local"

Reply:
Can we have some brief comments in the code to make this clear for everyone

Reply:
i can add that